release/ray_release/test_automation/ci_state_machine.py

```diff
@@ -31,8 +33,12 @@ def _move_hook(self, from_state: TestState, to_state: TestState) -> None:
         change = (from_state, to_state)
         if change == (TestState.PASSING, TestState.CONSITENTLY_FAILING):
             self._create_github_issue()
+            if os.environ.get("RAYCI_DISABLE_CI_TEST_BISECT") != "true":
```

Comment:
have a way to turn this off without a code change

Reply:
yess, this is temporary feature flag; i'll remove them 1, 2 weeks after the rolling out to production